lib/stripe_tester.rb

```diff
@@ -64,7 +64,7 @@ def self.post_to_url(data={})
       req = Net::HTTP::Post.new(post_url.path)
       req.content_type = 'application/json'
       req.body = data.to_json
-
+      req.basic_auth 'stripe', self.basic_authentication_password unless self.basic_authentication_password.nil?
```

Comment:
Good point! I forgot that `present?` was only available in Rails. But could you use something other than `unless` if you're going to use `nil?`, like `if !object.nil?`. I find it more readable :)